backend/spec/system/backoffice/auth_spec.rb

```diff
@@ -32,9 +32,9 @@
 
     it "works well" do
       visit "/backoffice"
-      expect(page).to have_text("Back office dashboard")
+      expect(page).to have_text("Project developers")
 
-      click_on "Sign Out"
+      click_on "Sign out"
```

Comment:
maybe when we already use localized text at app, we should also use it at tests :)